app/src/main/res/values-fr/strings.xml

```diff
@@ -97,7 +98,7 @@
     <string name="hectare">Hectare</string>
     <string name="sq_mile">Mile Carré</string>
     <string name="sq_yard">Yard Carré</string>
-    <string name="sq_foot">Pied carré</string>
+    <string name="sq_foot">Pied Carré</string>
```

Comment:
In French, only the first letter of the first word of a sentence should be capitalized, so I think it would be better to put the second word of each translation in lowercase (e.g. Mile Carré → Mile carré) instead of uppercase.

Reply:
Thanks for your suggestion. I agree with you and will edit all the other occurrences.
How about for the currencies? (ex: Dollar Australien) Should we follow the same logic?